business-wiki-frontend/pages/index.tsx

```diff
@@ -5,27 +5,28 @@ import buttonStyles from "../shared/styles/Button.module.scss";
 import Head from "next/head";
 import React, { FormEvent, useContext } from "react";
 import Input from "../shared/components/TextInput";
+import Footer from "../shared/components/Footer";
 import { useRouter } from "next/router";
 import { UserContext } from "../shared/contexts/UserContext";
-import { User } from "../shared/models/User";
+import { UserInterface } from "../shared/models/UserInterface";
 
 export default function App() {
```

Comment:
Destrukturyzacja.

Reply:
Resolved in 81ec19964319512cf05e726785768eb61b881f2c